src/test/java/org/solid/testharness/http/SolidClientTest.java

```diff
@@ -247,7 +248,7 @@ void getContainmentData() throws Exception {
     void getContainmentDataFails() throws IOException, InterruptedException {
         final Client mockClient = mock(Client.class);
         final URI resourceAcl = URI.create("http://localhost:3000/test");
-        final HttpResponse<String> mockResponse = mockStringResponse(500, null);
+        final HttpResponse<String> mockResponse = TestUtils.mockStringResponse(500, null);
```

Comment:
I'm not sure I understand this, since a proper Solid server shouldn't throw a 500, that would be an error with the server. It is certainly cases where it is appropriate for a mockup to do it, but I'm not too into the mockup :-)